bazel/p4c_deps.bzl

```diff
@@ -75,3 +75,35 @@ filegroup(
             strip_prefix = "protobuf-21.10",
             sha256 = "6fc9b6efc18acb2fd5fb3bcf981572539c3432600042b662a162c1226b362426",
         )
+    if not native.existing_rule("rules_foreign_cc"):
+        http_archive(
+            name = "rules_foreign_cc",
+            sha256 = "d54742ffbdc6924f222d2179f0e10e911c5c659c4ae74158e9fe827aad862ac6",
+            strip_prefix = "rules_foreign_cc-0.2.0",
+            url = "https://github.com/bazelbuild/rules_foreign_cc/archive/0.2.0.tar.gz",
```

Comment:
Nit: shall we try bumping this up to the latest version?

Reply:
The commands have changed in the new version. Get this errors:
```
ERROR: com_github_z3prover_z3/BUILD.bazel:8:15: @com_github_z3prover_z3//:api: no such attribute 'configure_env_vars' in 'configure_make' rule
ERROR: com_github_z3prover_z3/BUILD.bazel:8:15: @com_github_z3prover_z3//:api: no such attribute 'make_commands' in 'configure_make' rule
```
Could not figure out what needs to be changed how with quick search.

Reply:
Yes, +1, no need to dig deeper.